app/controllers/users_controller.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :load_user, only: [:edit,:show]
+  before_action :logged_in_user, except: %i[new create]
```

Comment:
```suggestion
  before_action :logged_in_user, except: %i(new create)
```
Dùng ngoặc tròn e nha, tương tự cho những chỗ còn lại